src/main/com/cqfn/degitx/simulator/Request.kt

```diff
@@ -2,12 +2,14 @@ package com.cqfn.degitx.simulator
 
 /**
  * Stub for Request sent via Network to Address.
+ * Data class.
```

Comment:
@IlyaMoskva you can use `data class` for data classes: https://www.geeksforgeeks.org/kotlin-data-classes/

Reply:
@g4s8 , you're right, there will not be any logic. Fixed.